config/taxonomy/Makefile

```diff
@@ -46,9 +46,9 @@ clean:
 	rm -f objects/taxonomy.json
 	rm -rf $(ROOT_DIR)/pkg/taxonomy/model/base
 	rm -rf $(ROOT_DIR)/pkg/connectors/openapiclient
-	mkdir -p $(ROOT_DIR)/connectors/tmp
-	cp  $(ROOT_DIR)/connectors/open_policy_agent/go/model.go  $(ROOT_DIR)/connectors/tmp
-	rm -rf $(ROOT_DIR)/connectors/open_policy_agent
-	mkdir -p $(ROOT_DIR)/connectors/open_policy_agent/go
-	cp  $(ROOT_DIR)/connectors/tmp/model.go $(ROOT_DIR)/connectors/open_policy_agent/go
-	rm -rf $(ROOT_DIR)/connectors/tmp
\ No newline at end of file
+#mkdir -p $(ROOT_DIR)/connectors/tmp
+#cp  $(ROOT_DIR)/connectors/open_policy_agent/go/model.go  $(ROOT_DIR)/connectors/tmp
```

Comment:
Instead of commenting I would suggest to remove it as we dont use the server target. An alternate approach would be to have a separate clean target only for server (maybe `clean-server`) which is called in `generate-server `target as a prerequisite.

Reply:
+1

Reply:
Incorporated the change in Makefile.